files/image_config/hostname/hostname-config.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Update hostname based on configdb
+Requires=database.service
+After=database.service
```

Comment:
how do we get the hostname in the first place in case of no dhcp. Some one will put the minigraph in the device, and then call config load-minigraph. Will that restart this hostname-config service to reconfigure the hostname?

Reply:
In current version of `config load_minigraph` it calls the commands in this bash script directly instead of restart this service. I can later modify sonic_utilities to simply restart this service instead.

Reply:
https://github.com/Azure/sonic-utilities/pull/154